### PR TITLE
Upadate from the upstream

### PR DESCRIPTION
**Pull request by obukhov**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,19 +1,41 @@
 {
-    "name": "lexpress/doctrine1",
+    "name": "friendsofsymfony1/doctrine1",
     "type": "library",
-    "description": "PHP5 Database ORM",
-    "keywords": ["orm", "database"],
+    "description": "PHP Database ORM for Symfony1. Do NOT use for new projects: please move to a newest Symfony release and Doctrine2",
```
Comment (terdia) [inline composer.json:4]: do not use for new project 😂

**Review by terdia — APPROVED** [review]
did you test against Mozart for bc?

**Comment by obukhov** [comment]
I am going to in php7.4 branch

File: composer.json (hunk 1 above)
Comment (obukhov) [reply composer.json:4]: I'm just gonna use it for new project, because nobody can tell what to do! 🤣

**Pull request merged**